### DISCUSSION TITLE
Using variables in MiKTeX

### DISCUSSION THREAD

**Discussion by russurquhart1** [MiKTeX/miktex#1465] [General]
Hi,

I'm still pretty new to TeX and MiKTex but I had a question. I want to be able to access a variable called VERSION that would store the current version number of my document and use that in my TeX document instead of having to hardcode the number each time. Also, I use the TeXworks application in MiKTeX to generate my PDF  versions of the documents. Is there a way, using TeXworks to have it also recognize this variable when generating the PDF?

Thanks for any help!

Russ

**Comment by LLdaniel** [comment]
Although this is not really a MikTeX question I will leave a short comment. 
You could define a variable in your `.tex` document like this:
```latex
\newcommand{\version}{v1.2.3}
```
and reference it with `\version` on other places in your document.
